__test__/pages/search/advancedSearchForm.test.tsx

```diff
@@ -73,6 +77,17 @@ describe("Advanced Search Form", () => {
       "/search?q=&filters%5BmaterialType%5D%5B0%5D=resourcetypes%3Anot&filters%5BmaterialType%5D%5B1%5D=resourcetypes%3Acar"
     )
   })
+  it("can check location checkboxes", async () => {
+    render(<AdvancedSearch isAuthenticated={true} />)
+    const location = searchAggregations.buildingLocation[0]
+    await userEvent.click(screen.getByLabelText(location.label))
+    submit()
+    // expect the label for notated music and cartographic
+    // ("resourcetypes:not", "resourcetypes:car") to be in url
```

Comment:
This isn't clear to me. Why is this true?

Reply:
oh thats an artifact from a test I copied! i will remove.